src/Web/Slack/Types.hs

```diff
@@ -35,7 +35,7 @@ import Web.Slack.Types.File
 import Web.Slack.Types.IM
 import Web.Slack.Types.Id
 import Web.Slack.Types.Item
-import Web.Slack.Types.Message
+import Web.Slack.Types.Message hiding (NewShortText)
```

Comment:
Because i want to allow creation of ShortText only by function "makeShort". It removes all symbols afer 30th and helps users to remember that this field should contain up to 30 symbols.

Reply:
In that case you just don't need to export `NewShortText` constructor from `Web.Slack.Types.Message` module.

Reply:
Ok, i ll hide it in  Web.Slack.Types.Message module.  I used this constructor in REPL when i was testing.

Reply:
For such cases people create functions like `unsafeMakeShort = NameOfTheConstructor`. `unsafe*` prefix is important here.

Reply:
Ok ! I ll do it next time =)

Reply:
I remember why i did it. Almost all functions and data types in Message.hs should be exported. That is why i did not write explicit export. But "NewShortText" should not be exported. Because i cant hide exporting, i hide it in import in reexport module.